common/src/test/java/tds/common/health/HealthIndicatorTest.java

```diff
@@ -0,0 +1,61 @@
+package tds.common.health;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.client.MockRestServiceServer;
+import org.springframework.web.client.RestTemplate;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.springframework.boot.actuate.health.Status.DOWN;
+import static org.springframework.boot.actuate.health.Status.UP;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.anything;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withSuccess;
+
+
+public class HealthIndicatorTest {
+    static RestTemplate restTemplate;
+    static HealthIndicatorClient client;
+    static MockRestServiceServer mockServer;
+
+    static String successBody = statusBody(UP.getCode());
+    static String failedBody = statusBody(DOWN.getCode());
+
+    static String statusBody(String text) {
+        return String.format("{\"status\" : \"%s\",  \"details\" : \"value\"}", text);
+    }
+
+    @BeforeClass
+    public static void before() {
```

Comment:
Why do we need this before class block and the other static properties?  I understand the non static Before is run before each test but does it matter that this is only configured once per class run?